source/index.md

```diff
@@ -2008,6 +2008,92 @@ The user can list all SoftLayer Cloud Server plans accessing the route bellow.
 
 Route: *`GET "/api/v1/shop-softlayer-cloud-server-plans"`*
 
+# User Shop Price
+
+When logged in, the user can access information about the shop. This access allows him to create shop prices.
+
+## Create
+
+> Create Shop Price
+
+```shell
+# INPUT
+curl --header "Content-Type: application/vnd.api+json" 
+  -H "Authorization: Bearer id-access-token" -X POST -d 
+'{ 
+  "data":
+  { 
+    "type":"shop-prices", 
+    "attributes":
+    { 
+      "components":
+      {
+        "disk-1-san":"25 GB (SAN)",
+        "disk-type":"san",
+        "datacenter":"wdc01",
+        "memory":"2 GB",
+        "processor":"4 x 2.0 GHz Cores",
+        "operating-system":"Debian - Latest (64 bit)",
+        "network-component":"1 Gbps Public & Private Network Uplinks"
+      },
+      "product":"cloud-server",
+      "provider":"softlayer"
+    }
+  }
+}' 
+http://acme.lvh.me:3000/api/v1/shop-price
+
+# OUTPUT
+{
+  "data":
+  {
+    "id":null,
+    "type":"shop-prices",
+    "links":
+    {
+      "self":"http://acme.lvh.me:3000/api/v1/shop-prices/"
+    },
+    "attributes":
+    {
+      "components":
+      {
+        "disk-1-san":"25 GB (SAN)",
+        "disk-type":"san",
+        "datacenter":"wdc01",
+        "memory":"2 GB",
+        "processor":"4 x 2.0 GHz Cores",
+        "operating-system":"Debian - Latest (64 bit)",
+        "network-component":"1 Gbps Public \u0026 Private Network Uplinks"
+      },
+      "hourly-price":"0.48",
+      "monthly-price":"300.0",
+      "product":"cloud-server",
+      "provider":"softlayer"
+    }
+  }
+}
+```
+
+```ruby
+Not implemented
+```
+
+The user can create a Shop Price accessing the route below.
+
+Route: *`POST "/api/v1/shop-price"`*
+
+Variable | Type | Value
+-------- | ---- | ----- 
```

Comment:
Would be better explain that attributes for some plans can be used, this way seems you can just check for cloud servers.